test/benchmark.js

```diff
@@ -0,0 +1,57 @@
+const { OatyArray } = require('../dist/index.js')
+const testArray = [{
+  a: 1,
+  b: 1,
+  fruit: 'apple'
+}, {
+  a: 1,
+  b: 2,
+  fruit: 'apple'
+}, {
+  a: 1,
+  b: 3,
+  fruit: 'banana'
+}]
+
+var results = []
+
+function time(runs) {
+    var oaty = new OatyArray({data: testArray});

```

Comment:
no code style enforced for test stuff atm but opting to not use semicolons - think [standardjs](https://github.com/standard)

Reply:
I read into ASI and you have to be more careful with JS. I don't think it's worth the risk and that semicolons make it abundantly clear.

Reply:
I've been writing TS/JS without semicolons for a while and haven't had any problems. I'm standing by this 😁

Reply:
Okay, I trust you

Reply:
Resolved